src/com/dotcms/rest/annotation/NoCache.java

```diff
@@ -0,0 +1,29 @@
+package com.dotcms.rest.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.dotcms.repackage.javax.ws.rs.NameBinding;
+/**
+Use: 
+	@NoCache
+	public Response indexCount ( @Context HttpServletRequest request, @PathParam ("query") String query,
+			@PathParam ("type") String type,
+			@PathParam ("callback") String callback ) throws DotDataException, DotSecurityException {
+
+
+		ResourceResponse responseResource = new ResourceResponse( paramsMap ); 
+
+		return responseResource.response( Long.toString( APILocator.getContentletAPI().indexCount( query, initData.getUser(), true ) ) );
+	}
+
+*/
+@NameBinding
+@Target({ ElementType.TYPE, ElementType.METHOD })
+@Retention(value = RetentionPolicy.RUNTIME)
+public @interface NoCache {
+	String cc() default "public, must-revalidate";
```

Comment:
Adding: 
String cc() default "no-cache, no-store, must-revalidate";
String expires() default "Mon, 07 Dec 2015 00:00:01 GMT";

here will allow the values to be overridden if required, while streamlining the filter slightly.